storey/drivers.py

```diff
@@ -1,17 +1,22 @@
 import base64
 import json
+import math
 import os
-from datetime import datetime, timedelta
-from typing import Optional
+import time
+from datetime import datetime, timedelta, timezone
+from enum import Enum
 from collections import OrderedDict
+from typing import List, Union, Optional
 import pandas as pd
 
+import redis
+import rediscluster
```

Comment:
We probably want to move these imports either to an inner scope or to another file, so that users (current and future) can use this library without installing redis (same idea as moving it to `extra-requires` as mentioned in another comment).

Reply:
Means the v3io imports should be moved as well? Where would you move it(what additional modules do you suggest)?

Reply:
This issue is still outstanding.

Reply:
done